client/logic/game/terrain/base.tscn

```diff
@@ -1,21 +1,16 @@
-[gd_scene load_steps=20 format=3 uid="uid://b4giy6a41fm5c"]
+[gd_scene load_steps=15 format=3 uid="uid://b4giy6a41fm5c"]
 
 [ext_resource type="Script" path="res://logic/game/terrain/terrain.gd" id="1_0oaqi"]
 [ext_resource type="Shader" uid="uid://bq8q1ltxmyovr" path="res://logic/shaders/color_shift.tres" id="2_a36y3"]
 [ext_resource type="PackedScene" uid="uid://2t61plm4vpda" path="res://logic/game/units/infantry.tscn" id="2_ugwn5"]
 [ext_resource type="PackedScene" uid="uid://d1vtobfji2rgv" path="res://logic/game/units/mech.tscn" id="3_3b5u4"]
 [ext_resource type="Texture2D" uid="uid://b4pxuxb2m8ko8" path="res://assets/images/game/tilemap_packed.png" id="3_5va3r"]
-[ext_resource type="PackedScene" uid="uid://cxq4g3bvxh5p3" path="res://logic/game/units/recon.tscn" id="4_0ggdr"]
 [ext_resource type="PackedScene" uid="uid://54war5m37akr" path="res://logic/ui/terrain_stats.tscn" id="4_icmkb"]
 [ext_resource type="PackedScene" uid="uid://c4iai7oi81hmi" path="res://logic/game/units/apc.tscn" id="5_xvo74"]
 [ext_resource type="PackedScene" uid="uid://jgjbc5qgpnh5" path="res://logic/game/units/artillery.tscn" id="6_57402"]
-[ext_resource type="PackedScene" uid="uid://dcy7mn6xmucyw" path="res://logic/game/units/tank.tscn" id="7_0duq7"]
 [ext_resource type="PackedScene" uid="uid://4464jjyowffi" path="res://logic/game/units/antiair.tscn" id="8_gmh13"]
 [ext_resource type="PackedScene" uid="uid://bss23v6r204mv" path="res://logic/game/units/missile.tscn" id="9_l4cvk"]
-[ext_resource type="PackedScene" uid="uid://ch51qqxsd8xsq" path="res://logic/game/units/rocket.tscn" id="10_x5naq"]
 [ext_resource type="PackedScene" uid="uid://dfki27p05gtry" path="res://logic/game/units/medium_tank.tscn" id="11_hohqx"]
-[ext_resource type="PackedScene" uid="uid://bn78onj8y8e6r" path="res://logic/game/units/piperunner.tscn" id="12_46ymp"]
-[ext_resource type="PackedScene" uid="uid://bhm7dlbtp7crw" path="res://logic/game/units/neo_tank.tscn" id="13_hiuf2"]
```

Comment:
why this resources are dropped from the scene?

Reply:
Those are shop units which get now generated by the game. It doesn't make sense to store them outside of the game. It is a leftover because the terrains have the tool mode. I've removed the export tag from the variable, which should make it volatile.